Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.yml

```diff
@@ -0,0 +1,180 @@
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'False'
+  description: If displaying similarity score for each fields.
+  isArray: false
+  name: showSimilarity
+  predefined:
+  - 'True'
+  - 'False'
+  required: false
+  secret: false
+- default: false
+  defaultValue: '0.2'
+  description: Filter out incident with a lower score than confidence.
+  isArray: false
+  name: confidence
+  required: false
+  secret: false
+- default: false
+  defaultValue: '100'
+  description: Maximun number of incidents to display
+  isArray: false
+  name: maxIncidentsToDisplay
+  required: false
+  secret: false
+- default: false
+  description: argument for the query
+  isArray: false
+  name: query
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'false'
+  description: Aggregate duplicate incidents within diffrerent dates. If includeIndicatorsSimilarity
+    is Truee then iaggreagateIncidentsDifferentDate will be False.
+  isArray: false
+  name: aggreagateIncidentsDifferentDate
+  predefined:
+  - 'True'
+  - 'False'
+  required: false
+  secret: false
+- default: false
+  defaultValue: '3000'
+  description: Limit of incident to query
+  isArray: false
+  name: limit
+  required: false
+  secret: false
+- default: false
+  description: 'Fields with json will be taken into account in order to compute similarity
+    - Example: CustomFields'
+  isArray: false
+  name: similarJsonField
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'False'
+  description: Use a predefined set of fields and CustomFields.
+  isArray: false
+  name: useAllFields
+  predefined:
+  - 'True'
+  - ' False'
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'False'
+  description: Show the incident you are investigating.
+  isArray: false
+  name: showActualIncident
+  predefined:
+  - 'True'
+  - 'False'
+  required: false
+  secret: false
+- default: false
+  description: Minimun number of indicators required related to the incident before
+    running the model.
+  isArray: false
+  name: minNumberOfIndicators
+  required: false
+  secret: false
+- default: false
+  description: Type of indicators we want to take into account  - If empty use all
+    indicators types.
+  isArray: false
+  name: indicatorsTypes
+  required: false
+  secret: false
+- default: false
+  description: Help to filter out indicators that appear in many incidents.
+  isArray: false
+  name: maxIncidentsInIndicatorsForWhiteList
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'False'
+  description: Include similarity of indicators from DBotFindSimilarIncidentsByIndicators
+    in final score.
+  isArray: false
+  name: includeIndicatorsSimilarity
+  predefined:
+  - 'True'
+  - ' False'
+  required: false
+  secret: false
+comment: Find similar incidents based on incident's fields similarity and indicators
```

Comment:
Comment is a a bit unclear with 2 usages of the word "similarity" 
Possible change: Find past incidents which share similar indicators with the current incident, where indicators similarity is calculated based on their fields similarity